apps/intellij/build.gradle.kts

```diff
@@ -194,3 +198,11 @@ tasks.register<DefaultTask>("publish") {
     group = "publish"
     description = "Placeholder task to workaround the semantic-release plugin"
 }
+
+fun buildCommands() = mutableListOf<String>().apply {
```

Comment:
I think there's no need of creating mutableListOf<String>() and use function apply

`fun buildCommands() = if(isOsWindows()) listOf("pwsh", "-command") else listOf("bash", "-c")`

if list should be mutable yo can replace listOf to mutableListOf

Reply:
Now this is just me being fairly new to Kotlin, but why do it this way instead of using `apply`? I'm just curious 😊

Reply:
Nothing wrong, more concise code :)